The Real Cost iOS App.xcodeproj/project.xcworkspace/contents.xcworkspacedata

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Workspace
+   version = "1.0">
+   <FileRef
+      location = "self:The Real Cost iOS App.xcodeproj">
+   </FileRef>
+</Workspace>
```

Comment:
To meet our coding standards you need an empty space at the end of this file

Reply:
Good idea, I forgot that.